### DISCUSSION TITLE
Add martial arts books to libraries

### DISCUSSION THREAD

**Discussion by SzQ1** [cataclysmbn/Cataclysm-BN#2044] [General]
Because I think they are not on the spawn list. I will do that later.

**Comment by Coolthulhu** [comment]
Should be rare drops.
Might be worth adding to mansions as well.

**Comment by SzQ1** [comment]
Well right now they are barely to be found, still should be possible to spawn in libraries, rare but possible.

**Comment by Dreamer1** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by pengwu26** [comment]
您好，您的邮件已收到，我会尽快给您答复！

**Comment by Krwak** [comment]
I second this. 
Maybe make it that you can't learn all the martial arts in the world from books but being able to find them would be nice.
Or make it that you can only swap between three different martial arts at a time that you have to 'prepare' before going to sleep or after sleeping and not being able to change otherwise.

**Comment by motzsascha515-jpg** [comment]
Sry dunno where to write this!can u pls ad to the martial arts mod the fist of the north star fighting style?i mean theres is panzerkunst too!;)
